API/controller/orderController.js

```diff
@@ -15,9 +30,91 @@ const getAllOrders = (req, res) => {
   });
 };
 
+// TO ADD AN ORDER
+const addOrder = (req, res) => {
+  const { error } = validateOrder(req.body);
+  if (error) {
+    return res.status(400).send(error.details[0].message);
+  }
+
+  const order = {
+    id: dummydb.orders.length + 1,
```

Comment:
You don't want to use the length of the array in determining the ids of your created objects. I'll recommend using [shortid](https://www.npmjs.com/package/shortid)